groups/sig-node/groups.yaml

```diff
@@ -54,6 +54,26 @@ groups:
       MembersCanPostAsTheGroup: "false"
       ReconcileMembers: "false"
 
+  - email-id: sig-node-ci@kubernetes.io
+    name: sig-node-ci
+    description: |-
+      SIG CI Working Group discussion group
+    owners:
+      - killen.bob@gmail.com # temporary to finish configuration
+      - sig-node-leads@kubernetes.io # SIG Node as a default owner
```

Comment:
A group cannot be an owner of another group
ref: https://support.google.com/a/answer/167094

Reply:
/lgtm cancel

Reply:
I thought this is the group: https://github.com/kubernetes/k8s.io/blob/0935943ff2b6c79bac7765a04d46584fa0341499/groups/groups.yaml#L19

It's where I took inspiration from

Reply:
fixed

Reply:
that one is a named account used by contribex for managing things

Reply:
yes, I was just confused by it - the name of is sounds like a group